packages.bib

```diff
@@ -18,8 +18,8 @@ @Manual{R-bcdata
 @Manual{R-bookdown,
   title = {bookdown: Authoring Books and Technical Documents with R Markdown},
   author = {Yihui Xie},
-  year = {2023},
-  note = {R package version 0.32},
+  year = {2022},
```

Comment:
packages should all be updated.  Its obvious none of them have been updated when looking at all these changes.